packages/kubernetes-1.20/kubelet-config

```diff
@@ -74,7 +74,7 @@ systemReserved:
   {{@key}}: "{{this}}"
   {{~/each}}
 {{~/if}}
-cpuManagerPolicy: "static"
+cpuManagerPolicy: {{default "static" settings.kubernetes.cpu-manager-policy}}
```

Comment:
When we originally rolled this out, we reverted it because changing the CPU manager policy caused issues on upgrade.

There's some local state that will need to be fixed up either as part of the migration, or as a helper command to check for a difference between the on-disk value and the configured value. The latter might make this setting possible to change on a running system, if still not entirely safe.